sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -0,0 +1,96 @@
+## API Report File for "@azure/schema-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export interface SchemaId {
+    // (undocumented)
+    id: string;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClient extends SchemaRegistryClientContext {
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+    getIdBySchemaContent(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetIdBySchemaContentResponse>;
```

Comment:
since both this API and `registerSchema` have the same four required string properties, I think I'd rather have this be an object shape:

```suggestion
    getIdBySchemaContent(schema: SchemaDescription, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetIdBySchemaContentResponse>;
```

and then elsewhere

```ts
interface SchemaDescription {
  groupName: string;
  schemaName: string;
  serializationType: string;
  schemaContent: string;
}

Reply:
I forwarded this feedback to the apiview for the Java client that is further along. We can match the outcome of that review when we build the outer layer over the generated code.